isis/src/base/objs/Spice/Spice.cpp

```diff
@@ -93,7 +80,7 @@ namespace Isis {
     Pvl &lab = *cube.label();
     PvlGroup kernels = lab.findGroup("Kernels", Pvl::Traverse);
     bool hasTables = (kernels["TargetPosition"][0] == "Table");
-    init(lab, !hasTables);
+    init(cube, !hasTables);
```

Comment:
Will this still work if I try to spiceinit a label in ISIS? I guess a better question, is the cube reading a superset of the label reading?

Reply:
Yes, this was mostly tested using label only.